cassandra/cqlengine/connection.py

```diff
@@ -43,7 +43,9 @@
 
 def format_log_context(msg, connection=None, keyspace=None):
     """Format log message to add keyspace and connection context"""
-    connection_info = connection if connection else DEFAULT_CONNECTION
+    connection_info = connection
+    if not connection_info:
```

Comment:
Could this be simplified to `connection_info = connection or 'DEFAULT_CONNECTION'`?